### ISSUE TITLE
[postgres] array_agg(ctid) returns null pointer exception

### ISSUE THREAD

**Issue by pyrocks** [dbeaver/dbeaver#6437]
Maybe related to https://github.com/dbeaver/dbeaver/issues/6312

#### System information: 
- Operating system (distribution) and version: mac os
- DBeaver version: 6.1.3

#### Connection specification:
- Database name and version: postgres 11.2
- Driver name: default
- Do you use tunnels or proxies (SSH, SOCKS, etc)? not relevant

#### Describe the problem you're observing:
array_agg(ctid) results in a nullPointerException in the value cell of the result grid

![image](https://user-images.githubusercontent.com/18515297/62455588-697c6380-b77f-11e9-8a33-9f339a3c6016.png)

#### Steps to reproduce, if exist:
```
SELECT array_agg(ctid)
FROM feed_contexts --some table
WHERE feed_id=5; --some condition returning more than 1 row
```

Note: the expression is perfectly valid - since the below code works:
```
do $$
declare
    aaa tid[];
begin
    SELECT array_agg(ctid)
    into aaa
    FROM feed_contexts
    WHERE feed_id=5;
    
    raise notice '%',aaa;
end; $$;
```

#### Include any warning/errors/backtraces from the logs

`java.lang.NullPointerException
    at org.jkiss.dbeaver.model.DBUtils.getEntityFromMetaData(DBUtils.java:1804)
    at org.jkiss.dbeaver.model.DBUtils.getEntityFromMetaData(DBUtils.java:1792)
    at org.jkiss.dbeaver.model.impl.jdbc.data.JDBCCollection.makeCollectionFromArray(JDBCCollection.java:197)
    at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCArrayValueHandler.getValueFromObject(JDBCArrayValueHandler.java:62)
    at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.getValueFromObject(PostgreArrayValueHandler.java:82)
    at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.getValueFromObject(PostgreArrayValueHandler.java:1)
    at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCComplexValueHandler.fetchColumnValue(JDBCComplexValueHandler.java:50)
    at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.fetchColumnValue(PostgreArrayValueHandler.java:49)
    at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCAbstractValueHandler.fetchValueObject(JDBCAbstractValueHandler.java:49)
    at org.jkiss.dbeaver.ui.controls.resultset.ResultSetDataReceiver.fetchRow(ResultSetDataReceiver.java:122)
    at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.fetchQueryData(SQLQueryJob.java:755)
    at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:484)
    at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$0(SQLQueryJob.java:407)
    at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:146)
    at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:405)
    at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:849)
    at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:2776)
    at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:98)
    at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:146)
    at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:96)
    at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:102)
    at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)`

**Comment by pyrocks** [comment]
I now see that array agg fails on every column, not just ctid. also workaround works. closing as duplicate.

**Comment by pyrocks** [comment]
confirm fixed in 6.1.4
